### PR TITLE
chore(docs): fix docs and mark some as deprecated

### PR DESCRIPTION
**Pull request by vpavlin**
# Description
First of all, sorry for a lot of changes in a single PR, but once I started...

I went through the `docs/` folder and fixed links to repo(s) and CI. I also fixed the doc factually in a few cases and marked some with `TODO: Depricate or fix` to get your opinion since they are no longer useful/working
 
# Changes

- [x] fix CI links: ci.status... -> ci.infra.status...
- [x] fir repo links `status-im/nim-waku` -> `waku-ord/nwaku` (no entirely necessary as Github will redirect, but it looks better from contributor/user perspective if the doc is not obviously outdated:) )
- [x] fix pointers to files (some moved with recent PR(s), som got renamed...)
- [x] add filter and lightpush v2 examples to waku.nimble and unify the example README

### CODE REVIEW

File: docs/contributors/waku-fleets.md [hunk 1]
```diff
@@ -20,8 +20,7 @@ and configured to bridge toy-chat messages to the `#waku channel` on the Vac Dis
 
 ### Fleet deployment rationale
 
-The `test` fleet is automatically updated after every commit to the `nim-waku` `master` branch
-and is therefore the most up to date representation of Waku v2 development.
+The `test` fleet is automatically updated after every commit to the `nim-waku` repository `master` branch and is therefore the most up to date representation of Waku v2 development.
```
Comment (jm-clius) [inline docs/contributors/waku-fleets.md:23]:
```suggestion
The `test` fleet is automatically updated after every commit to the `nwaku` repository `master` branch and is therefore the most up to date representation of Waku v2 development.
```

File: docs/tutorial/filter.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # Running Filter Protocol
 
+> TODO: Fix or depricate
+
```
Comment (jm-clius) [inline docs/tutorial/filter.md:4]: No strong opinion here, but perhaps we can just add the date and a note to say that this refers to an older version of nwaku and direct people to the operator guide? I think we may want to keep the record of old testnets and tutorials, though I'm open to just deleting these from the repo as well.

File: docs/contributors/waku-fleets.md (hunk 1 above)
Comment (vpavlin) [reply docs/contributors/waku-fleets.md:23]: Yeah, nim-waku is all over the place. I would say we can run the *.md files through `sed 's/nim-waku/nwaku/g` in a follow up PR?

File: examples/v2/README.md [hunk 3]
```diff
@@ -44,18 +46,16 @@ to the same pubsub and content topic.
 It runs forever, maintaining this subscription
 and printing the content it receives.
 
-**compile and run:**
+**Compile**
```
Comment (Ivansete-status) [inline examples/v2/README.md:50]:
```suggestion
**Run**
```
Did you mean `Run` instead:)?

**Review by Ivansete-status — APPROVED** [review]
That's great, thanks for it! Just a few comments

**Review by alrevuelta — APPROVED** [review]

File: docs/tutorial/dingpu.md [hunk 4]
```diff
@@ -19,7 +19,7 @@ Then type messages to publish.
 
 ## Interactively add a node
 
-There is also an interactive mode. Type `/connect` then paste address of other node. However, this currently has some timing issues with mesh not being updated, so it is adviced not to use this until this has been addressed. See https://github.com/status-im/nim-waku/issues/231
+There is also an interactive mode. Type `/connect` then paste address of other node. However, this currently has some timing issues with mesh not being updated, so it is adviced not to use this until this has been addressed. See https://github.com/waku-org/nwaku/issues/231
```
[the remark this answers is not in the document]
Comment (vpavlin) [reply docs/tutorial/dingpu.md:24]: Added TODO, thanks:)

**Pull request merged**